src/tweet.js

```diff
@@ -30,6 +30,7 @@ export default class Tweet {
       }
       Promise.all(promises)
         .then(() => {
+          this.getTransition().pause();
```

Comment:
@rodrigopavezi Curious to know how this solved the "Tweets showing for varied lengths of time" issue? Also, this negates the need for [line 35](https://github.com/Rise-Vision/rise-twitter/blob/0bc1b198007c9c4018db60fe73485601e7d7dbfa/src/tweet.js#L35) and makes it so that tweets update/restart even when it's in the middle of being played, doesn't it?

Reply:
@jungeunyoon when the module will send a twitter update on the middle of a transition if we don't pause it here it can start 2 setInterval with different starting time which causes the changing of tweets in different intervals. We could not stop it here and only start a transition if transition is paused. Thoughts? cheers

Reply:
@jungeunyoon also in line 35 it is checking viewer play and pause event and not transition event. They are two different things. You could change the playlist item schedule to play a twitter widget with a double of the time it would take to show the 25 tweets. Cheers